ops/services/web_application_firewall/main.tf

```diff
@@ -0,0 +1,36 @@
+resource "azurerm_web_application_firewall_policy" "sr_waf_policy" {
+  name                = "${var.name}-wafpolicy"
+  resource_group_name = var.resource_group_name
+  location            = var.resource_group_location
+
+  custom_rules {
+    name      = "US_Canada_Only"
```

Comment:
I know this is only in detection mode but we'd want to confirm that enforcing this restriction is ok. I would think we'd prefer to go through Akamai here.

Reply:
Do you have any info on whether that's turned on/integrated at the moment? We've brought that up in previous syncs, but I'm unsure if we ever closed the loop. At last check, I thought that was something we needed to engage the CDC on, but I may be mistaken!